aten/src/THC/generic/THCTensorMath.cu

```diff
@@ -256,6 +236,28 @@ void THCTensor_(catArray)(THCState *state, THCTensor *result,
   }
 }
 
+#if !defined(THC_REAL_IS_BOOL) /* non bool only part */
+
+void THCTensor_(zerosLike)(THCState *state, THCTensor *r_, THCTensor *input)
```

Comment:
this doesn't seem to be used.

Reply:
will be killed in https://github.com/pytorch/pytorch/pull/18201